### ISSUE TITLE
[web]  Camera focus

### ISSUE THREAD

**Issue by Dufayel** [juliansteenbakker/mobile_scanner#845]
In web version, the camera focus is not behaving correctly on some devices such as Galaxy S23, iPhone 13, and iPhone 15.

**Comment by navaronbracke** [comment]
Could you clarify this further? Perhaps with a video?

**Comment by Dufayel** [comment]
Hi @navaronbracke 

This test was done using an iPhone 15. When I bring the barcode close to the camera, it becomes blurry.

https://github.com/juliansteenbakker/mobile_scanner/assets/11709293/0ab4b57b-9a66-453e-b8d3-e1f820cb13a0

**Comment by navaronbracke** [comment]
And this works correctly on Android and iOS, right?

**Comment by mr-intj-22** [comment]
Hi @navaronbracke, I have the same issue on android. This issue happens only in the Front Camera. Tested on both 1+8T and a chinese 11.2" tablet, same issue on both, the camera can't focus on the barcode.

**Comment by navaronbracke** [comment]
Okay, then I'll investigate if this is an issue with ZXing or JsQr, as on native iOS/Android, the native camera is used. Maybe we do not setup the focus mode correctly on the web?

**Comment by Dufayel** [comment]
> And this works correctly on Android and iOS, right?

Yes, but it doesn’t happen on all devices, for example, it works perfectly on the iPhone 12.

**Comment by Dufayel** [comment]
I using this parameters for MobileScannerController

`final MobileScannerController cameraController = MobileScannerController(detectionSpeed: DetectionSpeed.noDuplicates, formats: [
    BarcodeFormat.upcA,
    BarcodeFormat.upcE,
    BarcodeFormat.ean8,
    BarcodeFormat.ean13,
    BarcodeFormat.code39,
    BarcodeFormat.code93,
    BarcodeFormat.code128,
  ]);`

**Comment by Dufayel** [comment]
Hi @navaronbracke, any update about it?

**Comment by navaronbracke** [comment]
Not yet, as I still need to investigate the issue further. I haven't had the time to look into it yet unfortunately.

**Comment by navaronbracke** [comment]
Apparently on the web, focus related settings are only available for image tracks and not video tracks.

See https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackSettings
